tools/run_mview_sperson_estimator.py

```diff
@@ -0,0 +1,217 @@
+                    view_idx = int(
+                        os.path.basename(img_list_path).rsplit(
+                            '.', 1)[0].split('_')[-1])
+                    logger.info(
+                        f'Visualizing keypoints3d from view {view_idx:02d}.')
+                    # visualize projected keypoints3d
+                    visualize_keypoints3d_projected(
+                        keypoints=keypoints3d,
+                        camera=cam_param_list[idx],
+                        output_path=os.path.join(
+                            scene_vis_dir,
+                            f'keypoints3d_pred_projected_{view_idx:02d}.mp4'),
+                        overwrite=True,
+                        plot_points=True,
+                        plot_lines=True,
+                        background_img_list=mview_img_list[idx],
+                        disable_tqdm=False,
+                        logger=logger)
+                    break  # TODO: remove this break
+            if smpl_data is not None:
+                body_model = mview_sp_smpl_estimator.smplify.body_model
+                for idx, img_list_path in enumerate(image_list_paths):
+                    view_idx = int(
+                        os.path.basename(img_list_path).rsplit(
+                            '.', 1)[0].split('_')[-1])
+                    logger.info(f'Visualizing {smpl_type.upper()}' +
+                                f' from view {view_idx:02d}.')
+                    with open(img_list_path, 'r') as f_read:
+                        lines = f_read.readlines()
+                    img_path = lines[0].strip()
+                    img_dir = os.path.join(args.data_root,
+                                           os.path.dirname(img_path))
+                    fisheye_param = cam_param_list[idx]
+                    pinhole_param = PinholeCameraParameter(
+                        K=fisheye_param.intrinsic,
+                        R=fisheye_param.extrinsic_r,
+                        T=fisheye_param.extrinsic_t,
+                        height=fisheye_param.height,
+                        width=fisheye_param.width,
+                        convention=fisheye_param.convention,
+                        world2cam=fisheye_param.world2cam,
+                        logger=logger)
+                    visualize_smpl_data(
+                        smpl_data=smpl_data,
+                        body_model=body_model,
+                        cam_param=pinhole_param,
+                        output_path=os.path.join(
+                            scene_vis_dir,
+                            f'{smpl_type}_data_projected_{view_idx:02d}.mp4'),
+                        overwrite=True,
+                        background_dir=img_dir,
+                        disable_tqdm=False,
+                        logger=logger,
+                        device=body_model.betas.device)
+                    break  # TODO: remove this break
```

Comment:
as above

Reply:
Removed in [83ac66b](https://github.com/openxrlab/xrmocap/pull/123/commits/83ac66b600c1681e720382df169657ac87ed3b88).